tests/supervisors/test_watchgodreload.py

```diff
@@ -11,7 +11,7 @@ def run(sockets):
     pass
 
 
-def test_watchgodreload(certfile_and_keyfile):
+def test_watchgodreload(tls_ca_certificate_pem_path, tls_ca_certificate_private_key_path):
```

Comment:
have to say I'm surprised I had to modify this since those fixtures are not used in the test, I think we shoud remove them, letting it that way just in case I missed something